packages/core/src/transport/WsOutboundTransport.ts

```diff
@@ -68,18 +68,14 @@ export class WsOutboundTransport implements OutboundTransport {
     const isNewSocket = !this.hasOpenSocket(socketId)
     const socket = await this.resolveSocket({ socketId, endpoint, connectionId })
 
-    return new Promise<void>((resolve, reject) =>
-      socket.send(Buffer.from(JSON.stringify(payload)), (err) => {
-        // If the socket was created for this message and we don't have return routing enabled
-        // We can close the socket as it shouldn't return messages anymore
-        if (isNewSocket && !outboundPackage.responseRequested) {
-          socket.close()
-        }
-
-        if (err) return reject(err)
-        resolve()
-      })
-    )
+    // If the socket was created for this message and we don't have return routing enabled
+    // We can close the socket as it shouldn't return messages anymore
+    // make sure to use the socket in a manner that is compliant with the https://developer.mozilla.org/en-US/docs/Web/API/WebSocket
+    // (React Native) and https://github.com/websockets/ws (NodeJs)
+    socket.send(Buffer.from(JSON.stringify(payload)))
```

Comment:
We technically could send just a string instead of a buffer with both these interfaces, but it should not matter that much.

Reply:
Is the method sync in RN? And if we don't provide a callback, is it sync in NodeJS as well?

Reply:
Haven't checked it, but there is likely an overload to not provide a callback as you don't have to handle the res/err if you don't want. But yeah it would be good to do a test run with this in both environments.

Reply:
I think this should be fine. It's also the behavior that was there up until 2-3 weeks ago.